src/Symfony/Component/Security/Core/Encoder/NativePasswordEncoder.php

```diff
@@ -59,33 +59,38 @@ public function __construct(int $opsLimit = null, int $memLimit = null, int $cos
      */
     public function encodePassword($raw, $salt): string
     {
-        if (\strlen($raw) > self::MAX_PASSWORD_LENGTH) {
+        if (\strlen($raw) > self::MAX_PASSWORD_LENGTH || (PASSWORD_BCRYPT === $this->algo && 72 < \strlen($raw))) {
```

Comment:
given `PASSWORD_DEFAULT !== PASSWORD_BCRYPT` as of php 7.4 (`null` const), should we use `PASSWORD_BCRYPT` instead when setting `$this->algo` in the constructor? 
Btw now that `PASSWORD_*` consts are strings, using `max()` to determine the best algo might be wrong at some point

Reply:
> should we use PASSWORD_BCRYPT instead when setting $this->algo in the constructor

nope we shouldn't because the purpose in the constructor is to select the best available algo, which is not bcrypt anymore already.

> using max() to determine the best algo might be wrong at some point

good catch, I'll fix that on 4.3!